dogfooding/lib/screens/login_screen.dart

```diff
@@ -237,3 +246,104 @@ String randomId({int size = 21}) {
   }
   return id;
 }
+
+class EnvironmentSwitcher extends StatefulWidget {
+  const EnvironmentSwitcher({
+    super.key,
+    required this.currentEnvironment,
+  });
+
+  final Environment currentEnvironment;
+
+  @override
+  State<EnvironmentSwitcher> createState() => _EnvironmentSwitcherState();
+}
+
+class _EnvironmentSwitcherState extends State<EnvironmentSwitcher> {
+  late Environment selectedEnvironment;
+
+  @override
+  void initState() {
+    selectedEnvironment = widget.currentEnvironment;
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Row(
+        children: [
+          Text(
+            'Current environment: ${selectedEnvironment.displayName}',
```

Comment:
I think this is too verbose and feels out of place with the rest of the polished UI. How about making it a banner or a ribbon (like the debug one) with just the name of the environment?